ara/ui/templates/base.html

```diff
@@ -9,25 +9,20 @@
   <meta name="author" content="https://github.com/ansible-community/ara">
   <title>ARA Records Ansible</title>
 
-  {% static 'images/favicon.ico' as favicon_url %}
-  <link rel="shortcut icon" href="{% static_url favicon_url %}">
-  {% static 'css/ara.css' as ara_css_url %}
-  <link rel="stylesheet" href="{% static_url ara_css_url %}">
-  {% static UI_THEME as ui_theme_url %}
-  <link rel="stylesheet" href="{% static_url ui_theme_url %}">
-  {% with "css/pygments-"|add:UI_THEME_VARIANT|add:".css" as pygments_css %}
-    {% static pygments_css as pygments_css_url %}
-    <link rel="stylesheet" href="{% static_url pygments_css_url %}">
-  {% endwith %}
+  <link rel="shortcut icon" href="{% static 'images/favicon.ico' %}">
+  <link rel="stylesheet" href="{% static 'css/ara.css' %}">
+  <link id="bootstrap-dark-css" rel="stylesheet" href="{% static 'css/bootstrap-darkly.min.css' %}" disabled>
+  <link id="pygments-dark-css" rel="stylesheet" href="{% static 'css/pygments-dark.css' %}" disabled>
+  <link id="bootstrap-light-css" rel="stylesheet" href="{% static 'css/bootstrap-default.min.css' %}">
```

Comment:
As discussed we should probably replace the default bootstrap by [flatly](https://bootswatch.com/flatly/) so both themes are based off of bootswatch.

Reply:
understood, I will fix this